.gitmodules

```diff
@@ -36,3 +36,7 @@
 	path = pk
 	url = https://github.com/riscv-software-src/riscv-pk.git
 	branch = master
+[submodule "llvm"]
+	path = llvm
+	url = https://github.com/llvm/llvm-project.git
+	branch = release/15.x
```

Comment:
Is there a reason to not go to `16.x`?

Reply:
Let move to https://github.com/riscv-collab/riscv-gnu-toolchain/pull/1263 :) I guess the only reason is the PR was created in last year which LLVM 16 isn't release yet.